api/src/pcapi/core/finance/api.py

```diff
@@ -1936,21 +1936,33 @@ def _prepare_invoice_context(invoice: models.Invoice, use_reimbursement_point: b
         groups.append(invoice_group)
     reimbursements_by_venue = get_reimbursements_by_venue(invoice)
 
+    reimbursement_point = None
+    reimbursement_point_name = None
+    reimbursement_point_iban = None
+
     # FIXME (dbaty, 2022-07-18): once business units are not used
     # anymore, stop using `venue` in the template.
     if use_reimbursement_point:
         reimbursement_point = invoice.reimbursementPoint
+        if reimbursement_point:
+            reimbursement_point_name = reimbursement_point.publicName or reimbursement_point.name
+            bank_information = offerers_repository.BankInformation.query.filter(
+                offerers_repository.BankInformation.venueId == reimbursement_point.id
+            ).one_or_none()
```

Comment:
```suggestion
            ).one()
```
Comme plus haut, le cas `None` ne me semble pas normal.